content/shmem_team_translate_pe.tex

```diff
@@ -11,20 +11,23 @@
 \end{Csynopsis}
 
 \begin{apiarguments}
-\apiargument{IN}{src\_team}{A valid SHMEM team handle.}
-\apiargument{IN}{src\_pe}{A \ac{PE} number in src\_team.}
-\apiargument{IN}{dest\_team}{A valid SHMEM team handle.}
+\apiargument{IN}{src\_team}{An \openshmem team handle.}
+\apiargument{IN}{src\_pe}{A \ac{PE} number in \VAR{src\_team}.}
+\apiargument{IN}{dest\_team}{An \openshmem team handle.}
 \end{apiarguments}
 
 \apidescription{
 The \FUNC{shmem\_team\_translate\_pe} routine will translate a given \ac{PE} number
-to the corresponding \ac{PE} number in another team.
+in one team into the corresponding \ac{PE} number in another team.
 Specifically, given the \VAR{src\_pe} in \VAR{src\_team}, this routine returns that
 \ac{PE}'s number in \VAR{dest\_team}. If \VAR{src\_pe} is not a member of both the
 \VAR{src\_team} and \VAR{dest\_team}, a value of \CONST{-1} is returned.
 
-
-If either of the \VAR{src\_team} or \VAR{dest\_team} handle is invalid, the behavior is undefined.
+If at least one of \VAR{src\_team} and \VAR{dest\_team} compares equal
+to \LibConstRef{SHMEM\_TEAM\_INVALID}, then a nonzero value is
+returned.
+If either of the \VAR{src\_team} or \VAR{dest\_team} handles are
```

Comment:
Should we replace "nonzero value" with "-1" when at least one of the teams is `SHMEM_TEAM_INVALID`? At return value description, it says
```
or a value of -1 if any team handle arguments are invalid or the src_pe is not in both the source and destination teams.
```